edx_event_bus_kafka/management/commands/produce_event.py

```diff
@@ -29,9 +29,8 @@ class Command(BaseCommand):
                     "id": 123,
                     "is_active": true,
                     "pii": {"username": "foobob", "email": "bob@foo.example", "name": "Bob Foo"}}}'
-
-    # TODO (EventBus): Potentially update example topic and group names to follow any future naming conventions.
     """
+    # TODO: Potentially update example topic and group names to follow any future naming conventions.
```

Comment:
Can we just get our current topic and group name and determine if the example should be updated or not, and close this?

Reply:
Do you mean that we should use a real in-use topic and group name here?

...actually, I just realized that there's an error here and in the first line of the docstring -- mentions of consumers and groups are both incorrect.

Reply:
I think what you did here works fine.